examples/3D_weak_scaling/case.py

```diff
@@ -199,7 +199,7 @@
     'mpp_lim'                      : 'T',
     'mixture_err'                  : 'T',
     'time_stepper'                 : 3,
-    'weno_order'                   : 3,
+    'weno_order'                   : 5,
```

Comment:
why the change?

Reply:
Ben had done it to examine profiles for WENO5, I will revert back to original

Reply:
This change was made because the performance results on NVIDIA hardware we were comparing to use WENO5 and this is the case we were using for performance results on AMD hardware.